openstack/grafana/values.yaml

```diff
@@ -41,6 +41,9 @@ postgres:
     os_password: DEFINED-IN-REGION-SECRETS
 persistence:
   enabled: false
+  name: grafana-prometheus-storage
```

Comment:
name should be `storage-grafana-postgres-0` if you want to use the existing one.